fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/node/CoalesceExpNode.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.database.utils.query.node;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Stack;
+
+import com.ibm.fhir.database.utils.query.expression.ColumnRef;
+
+
+/**
+ * The COALESCE SQL function
+ */
+public class CoalesceExpNode implements ExpNode {
```

Comment:
Ok - I get this class,  it has  a call hierarchy... the other coalesce appears like it won't be used.